batchflow/pipeline.py

```diff
@@ -946,69 +946,81 @@ def m(self, name, batch=None):
         """ A shorter alias for get_model_by_name() """
         return self.get_model_by_name(name, batch=batch)
 
-    def init_model(self, mode, name=None, model_class=None, config=None):
+    def init_model(self, name, model_class=None, mode='dynamic', config=None):
```

Comment:
Having a default `name='model'` (here and in other methods) would be nice, as most pipelines contain exactly one model

Reply:
That would require more letters to type:
`.init_model('model', ResNet18)`
`.init_model(model_class=ResNet18)`

Reply:
Do we still need `mode`?